src/modules/ROOT/pages/changelog/202209/02.json

```diff
@@ -0,0 +1,491 @@
+    {
+      "what": [
+        "the uninformative RBAC rule import error messages reported by the provisioning system.",
+        "The error was uninformative because RBAC rules, in this context, don't use identifiers."
+      ],
+      "why": [
+        "because the error in question was uninformative and looked like this: `failed to complete transaction: store encoder encode rbac-rule []: unsupported resource type '' for RBAC store encode`."
+      ],
+      "how": [
+        "by allowing custom error reporting logic for specific resources such as RBAC rules."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/e2ac9c4f3e5ddc2a7e2449bf7a5911774cefa451"
+      ]
+    },
+    {
+      "what": [
+        "password constraints which saved their values as strings to save them as numbers.",
+        "Additionally, the labels of the two fields were reworded to match the style of the other inputs in the password constraints section."
+      ],
+      "why": [
+        "because the password constraints were saved as strings; with this change, the constraints will be saved as numbers."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-locale/commit/703516c9e23bf5373de01ba073b11426cbe66390",
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/aad919f90700a5b8de8f15c299ddf351f3c2e1b2"
+      ]
+    },
+    {
+      "what": [
+        "the return type of the password constraint fields and a bit of rewording for upper and lowercase constraints."
+      ],
+      "why": [
+        "because the password constraints are numbers and should be saved as such, not strings.",
+        "The rewording was done to match the consistency of the other password constraints inputs."
+      ],
+      "how": [
+        "by changing the password constraint's input to work with numbers and correcting the translation strings."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-locale/commit/703516c9e23bf5373de01ba073b11426cbe66390",
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/a2480009e6f1c59c10569343bef98270ce0968d2"
+      ]
+    },
+    {
+      "what": [
+        "the documentation about sanitizers and fixed the example links in module field validation tab to now point to the correct anchors."
+      ],
+      "why": [
+        "the documentation was lacking information about field value sanitization."
+      ],
+      "how": [
+        "by updating the documentation with a new important section describing this behavior and changing the documentation links in the module field configuration to point to validators/sanitizers."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/9af28f269ae4752c036522273ec9c1af6a14efd2"
+      ]
+    },
+    {
+      "what": [
+        "page tab when a record was being created/edited/viewed, the operation was reflected in the title of the public page."
```

Comment:
Can you doublecheck this one; I reworded it a bit but both the original and the new one don't make much sense to me